instrumentation/jsp-2.3/javaagent/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ muzzle {
   pass {
     group.set("org.apache.tomcat")
     module.set("tomcat-jasper")
-    // version range [7.0.0,7.0.19) is missing from maven
+    // version range [7.0.0,7.0.19) is missing from maven, can't add assertInverse
```

Comment:
`can't add assertInverse` isn't really helpful as it does not provide any information on why it can't be added

Reply:
> `can't add assertInverse` isn't really helpful as it does not provide any information on why it can't be added

I elaborated it. How about now?